client/src/components/Chart.jsx

```diff
@@ -1,103 +1,54 @@
-    type: chartType,
-    data: {
-      labels: temperatureData.labels,
-      datasets: [
-        {
-          label: 'Patient temperature',
-          backgroundColor: '#4760ff',
-          data: temperatureData.values,
-        },
-      ],
-    },
-    options: {
-      scales: {
-        yAxes: [
-          {
-            ticks: {
-              max: 110,
-              min: 80,
-            },
-          },
-        ],
-      },
-    },
+  const { observations } = props;
+
+  const temps = observations.map(observation => {
+    const datetime = new Date(observation.date).toISOString().slice(0, 10);
+    let temp;
+    if (
+      !observation.physical.feverSeverity ||
+      observation.physical.feverSeverity === '' ||
+      !parseFloat(observation.physical.feverSeverity, 10) > 0
+    ) {
+      temp = 0;
+    } else {
+      temp = parseFloat(observation.physical.feverSeverity, 10);
+    }
+
+    return { date: datetime, temperature: temp };
   });
 
-  const chartContainer = useRef(null);
-
-  const updateChartConfig = useCallback(() => {
-    const map = new Map();
-    observations.map(observation => {
-      const date = new Date(observation.date).toISOString().slice(0, 10);
-
-      let temp;
-      if (
-        !observation.physical.feverSeverity ||
-        observation.physical.feverSeverity === '' ||
-        !parseInt(observation.physical.feverSeverity, 10) > 0
-      ) {
-        temp = 0;
-      } else {
-        temp = parseInt(observation.physical.feverSeverity, 10);
-      }
-
-      if (map.has(date)) {
-        if (map.get(date).temp !== 0) {
-          map.get(date).temp = (map.get(date).temp + temp) / (map.get(date).numObservations + 1);
-        }
-        map.set(temp, {
-          temp,
-          date,
-        });
-      } else {
-        map.set(date, {
-          temp,
-          date,
-        });
-      }
-      return true;
-    });
-
-    const dates = [];
-    const temps = [];
-    map.forEach(entry => {
-      dates.push(entry.date);
-      temps.push(entry.temp);
-    });
-
-    setChartConfig(config => {
-      const newConfig = config;
-      newConfig.data.labels = dates;
-      newConfig.data.datasets[0].data = temps;
-      return newConfig;
-    });
-  }, [observations]);
-
-  useEffect(() => {
-    updateChartConfig();
-    // eslint-disable-next-line no-unused-vars
-    const chart = new chartJs.Chart(chartContainer.current, chartConfig);
-  }, [chartContainer, chartConfig, updateChartConfig]);
+  const [temperatureData] = useState(temps);
 
   return (
-    <div>
-      <canvas id="chart" ref={chartContainer} />
-    </div>
+    <ResponsiveContainer width="100%" aspect={4.0 / 1.5}>
+      <LineChart
+        data={temperatureData}
+        margin={{
+          top: 10,
+          right: 50,
+          left: 20,
+          bottom: 5,
+        }}
+      >
+        <CartesianGrid strokeDasharray="3 3" />
+        <XAxis dataKey="date" />
+        <YAxis type="number" domain={[90, 110]} />
```

Comment:
Just to make this a little bit narrower so the chart fills as much of the screen as we can, I'd suggest maybe making this like [94, 105] - if someone is less than 94 or higher than 105 they really should be already in a hospital lol